tests/integration/test_tree.py

```diff
@@ -20,14 +21,16 @@ def test__get_tree_url():
         assert "tree.json" in tree_url_parts.path
 
 
-def test__get_tree_url_bad_date():
+def test__get_tree_url_bad_date(patch_s3_for_tests):
+    # patch_s3_for_tests: Mocks S3 sequence data to prevent failures from missing historical versions
     # we cannot get reference trees prior to 2024-08-01
     ct = CladeTime()
     with pytest.raises(TreeNotAvailableError):
         Tree(datetime(2024, 7, 13, tzinfo=timezone.utc), ct.url_sequence)
 
 
-def test_tree_ncov_metadata():
+def test_tree_ncov_metadata(patch_s3_for_tests):
+    # patch_s3_for_tests: Mocks S3 sequence data to prevent failures from missing historical versions
```

Comment:
remove the patch if possible, even if it means re-coding hard-coded values. keep it simple!